test/JDBC/expected/BABEL-5119-vu-verify.out

```diff
@@ -0,0 +1,186 @@
+-- tsql
+CREATE LOGIN login_babel5119_1 WITH PASSWORD = '12345678'
+CREATE LOGIN login_babel5119_2 WITH PASSWORD = '12345678'
+ALTER authorization on database::BABEL5119_1 TO login_babel5119_1
```

Comment:
Add db or something similar in database name to make it recognizable.

Reply:
Changed `BABEL5119_1` to `BABEL5119_db`